controllers/bpfman-operator/configmap.go

```diff
@@ -209,8 +229,28 @@ func bpfmanConfigPredicate() predicate.Funcs {
 	}
 }
 
+// LoadRestrictedSecurityContext loads the bpfman-restricted SCC from disk which
+// users can bind to in order to utilize bpfman in an unprivileged way.
+func LoadRestrictedSecurityContext(path string) *osv1.SecurityContextConstraints {
+	// Load static SCC yaml from disk
+	file, err := os.Open(path)
```

Comment:
why u decided to load static file instead of setting scc obj using APIs ?

Reply:
It's just how we've been doing it for all of the objects deployed by the bpfman-operator (i.e the `bpfman-daemonset`, `csiDriver`, and now `SecurityContextConstraint`) and  IMO it usually ends up being a bit less code

Reply:
Additionally they all really didn't need to be configured at runtime at all (except for the bpfman ds) so I figured having them static on disk made sense